recipes/ncl/build.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/sh
+
+export CC=${PREFIX}/bin/gcc
+export CXXFLAGS="-fPIC"
+export LDFLAGS="-L${PREFIX}/lib"
+export CPPFLAGS="-I${PREFIX}/include"
+export CFLAGS="-I${PREFIX}/include"
+
+if [ "$(uname)" = "Darwin" ]; then
+    if [ -d "/opt/X11" ]; then
+        x11_lib="-L/opt/X11/lib"
+        x11_inc="-I/opt/X11/include -I/opt/X11/include/freetype2"
+    else
+        echo "No X11 libs found. Exiting..." 1>&2
+        exit
+    fi
+
+    conf_file=config/Darwin_Intel
+elif [ "$(uname)" = "Linux" ]; then
+    conf_file=config/LINUX
+fi
+
+mkdir triangle_tmp && cd triangle_tmp && curl -q http://www.netlib.org/voronoi/triangle.shar | sh && mv triangle.? ../ni/src/lib/hlu/. && cd -
+
+# add "-std=c99" to compile config files -- not needed after NCL 6.3.0
+sed -e "s/^\(#define CcOptions.*\)$/\1 -std=c99/" -i.backup "${conf_file}"
+
+# install csh on CircleCI builds
+if [ "${CIRCLE_BUILD_IMAGE}" = "ubuntu-12.04" ]; then
+    sudo apt-get install csh
```

Comment:
The `CircleCI` docker image we use is CentOS, so `apt-get` will not work. Also we do not use `sudo` for safety reasons and to allow people to run the recipes locally.

To install `csh` you need to create a file called `yum_requirements.txt` and add `csh` there. The `conda-smithy` will do the right thing and install that for you.